src/utilities/util_verify.c

```diff
@@ -23,8 +23,11 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
     dump_address = dump_blocks = dump_history = dump_layout = dump_pages = hs_verify =
       stable_timestamp = false;
     config = dump_offsets = uri = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "d:hS")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "ad:s")) != EOF)
         switch (ch) {
+        case 'a':
```

Comment:
Happy to use another flag instead of "-a". "-h" implies home_dir and "-d" is already taken. "-a" stands for archive, but happy to use anything else. I have added a description to these options in the usage(), to add clarity to what these options mean.